client/blocks/reader-featured-video/style.scss

```diff
@@ -1,3 +1,16 @@
 .reader-featured-video__video iframe {
 	display: flex;
 }
+
+.reader-featured-video__play-icon {
+	opacity: .8;
```

Comment:
You're right - the WP CSS coding standard mandates a leading zero.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/

We're using a mixture of with- and without-leading-zero across Calypso though, I notice.

I'll add a `0` :)